test/chtest-make-auto

```diff
@@ -17,21 +33,21 @@ tag_from_path () {
     fi
 
     case $basicname in
-    Build|Dockerfile|Docker_Pull)
-        if [[ $parent_dir == '' ]]    || \
-           [[ $parent_dir == '.' ]]   || \
-           [[ $parent_dir == test ]]; then
+    Build|Dockerfile)
+        if    [[ $dir == '' ]]  \
+           || [[ $dir == '.' ]] \
+           || [[ $dir == test ]]; then
```

Comment:
`[[ ... ]]` should put OR and AND operators within the brackets, `[ ... ]` outside.